metricbeat/module/docker/cpu/data.go

```diff
@@ -0,0 +1,39 @@
+package cpu
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	//"fmt"
+)
+
+func eventsMapping(cpuStatsList []CPUStats) []common.MapStr {
+	myEvents := []common.MapStr{}
+	//fmt.Printf(" Taille cpuStatsList : ",len(cpuStatsList),"\n")
+	for _, cpuStats := range cpuStatsList {
+		myEvents = append(myEvents, eventMapping(&cpuStats))
+	}
+	//fmt.Printf(" Taille events : ",len(myEvents),"\n")
+	return myEvents
+}
+func eventMapping(mycpuStats *CPUStats) common.MapStr {
+
+	//fmt.Printf(" From data : Nom du container : ",mycpuStats.MyContainer.Name,"\n")
+	//fmt.Printf(" ID du container : ",mycpuStats.MyContainer.Id,"\n")
+
+	event := common.MapStr{
```

Comment:
Check our new method for parsing and normalizing fields: https://www.elastic.co/guide/en/beats/metricbeat/current/creating-metricsets.html#_parsing_and_normalizing_fields We just added this very recently.